dapps/digital-twin-detail/src/components/container/List.ts

```diff
@@ -151,4 +151,8 @@ export default class ContainerListComponent extends mixins(EvanComponent) {
 
     (this.$refs.listItemDetail as any).showPanel();
   }
+
+  onShare(): void {
+    (this.$refs.shareContainer as any).showPanel();
```

Comment:
can you try if `as Vue` also works?

Reply:
We never declare it on the Vue instance, so it can't work. I imported the swipepanel now and it works :)